hummingbot/connector/exchange/zebpay/zebpay_resolve.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+# API Feed adjusted to sandbox url
+from hummingbot.core.event.events import TradeType, OrderType
+from hummingbot.core.utils.asyncio_throttle import Throttler
+
+
+_ZEBPAY_REST_URL_SANDBOX = "https://www.zebpay.co/pro/v1"
+_ZEBPAY_REST_URL_PROD = "https://www.zebapi.com/pro/v1"
```

Comment:
Maybe we should not include the `/pro/v1` part in the base url, because of (as you showed me) some routes use `/api/v1` and others `/pro/v1`.

Reply:
agree - though I'm unsure where the `/api/v1` part is used. Is in the User API requests?